modules/test/playwright/tests/object-web/objectEntries.spec.ts

```diff
@@ -169,6 +169,10 @@ test.describe('Manage object entries through Page Templates', () => {
 				page.getByRole('menuitem', {exact: true, name: itemValue})
 			).toBeVisible();
 		});
+
+		// Change portal back to english language
+
+		await page.goto('/en');
```

Comment:
i think this should be inside an afterEach function, like the one in `listTypeDefinitions.spec.ts`

Reply:
Hey @rebsilva! Just applied the afterEach! I implemented only the siteLanguage condition, since it's enought in this case. Please let me know what you think about, thanks!!